src/seedu/addressbook/data/person/Person.java

```diff
@@ -15,16 +18,24 @@
     private Email email;
     private Address address;
 
-    private final UniqueTagList tags;
+    private final Set<Tag> tags;
+
     /**
      * Assumption: Every field must be present and not null.
      */
-    public Person(Name name, Phone phone, Email email, Address address, UniqueTagList tags) {
+    public Person(Name name, Phone phone, Email email, Address address, Set<Tag> tags) {
         this.name = name;
         this.phone = phone;
         this.email = email;
         this.address = address;
-        this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.tags = tags;
```

Comment:
Should be:

```java
this.tags = new HashSet<>(tags);
```

so that the `Person` will have a `Set` which it has full ownership of, and can't be modified externally (preserves encapsulation).

Can copy from AB-4:

https://github.com/se-edu/addressbook-level4/blob/e186102b646d2307b53928142dd22932a6f4c86d/src/main/java/seedu/address/model/person/Person.java#L25-L37